packages/nextra-theme-blog/src/mdx-theme.js

```diff
@@ -136,7 +136,7 @@ const A = ({ children, ...props }) => {
   const isExternal = props.href && props.href.startsWith('https://')
   if (isExternal) {
     return (
-      <a target="_blank" {...props}>
+      <a target="_blank" rel="noopener noreferrer" {...props}>
```

Comment:
```suggestion
      <a target="_blank" rel="noopener" {...props}>
```

Generally just need `noopener`, as referrer can be useful for referrals!

Reply:
Works for me!